ui/js/controllers/perf/graphs.js

```diff
@@ -547,7 +547,7 @@ perf.controller('GraphsCtrl', [
             $state.transitionTo('graphs', {
                 series: $scope.seriesList.map(function (series) {
                     return "[" + [series.projectName,
-                        series.signature,
+                        series.id,
                         (series.visible ? 1 : 0),
                         series.frameworkId] + "]";
```

Comment:
I realized that you actually don't need the "[" braces (we automatically discard them), so we can just go ahead and remove them. This might actually be better written as:

```js
$scope.seriesList.map(series => `${series.projectName},${series.id},${series.visible ? 1 : 0},${series.frameworkId}`)
```